tests/PHPStan/Analyser/data/bug-4890.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Bug4890;
+
+use function PHPStan\Testing\assertType;
+
+interface Proxy {}
+
+class HelloWorld
```

Comment:
see https://phpstan.org/r/b6b95d5a-a3fb-4c29-be47-4e7832f00ddc regarding how this test fails before this PR